lib/index.js

```diff
@@ -46,31 +38,24 @@ exports.plugin = {
     }
 
     const analytics = new Analytics(options)
-
     server.decorate('request', 'ga', request => analytics.ga(request), { apply: true })
-
     server.ext('onPreResponse', async (request, h) => {
+      const allowTrackAnalytics = await options.trackAnalytics(request)
+      const shouldTrack = (await options.trackAnalytics()) ? allowTrackAnalytics : false
```

Comment:
You're calling options.trackAnalytics twice. I suspect this is a mistake - this used to be a check for whether trackAnalytics existed and then to call it if it did.
Also, it was set to track by default because this is the expected behaviour - IWTF is not the only service to use this library, so maintaining the existing functionality for other clients is important. Please change this back to track by default, otherwise you'll be altering hapi-gapi for other libraries.